ghproxy/ghcache/coalesce.go

```diff
@@ -134,6 +136,14 @@ func (r *requestCoalescer) RoundTrip(req *http.Request) (*http.Response, error)
 	ghmetrics.CollectCacheRequestMetrics(string(cacheMode), req.URL.Path, req.Header.Get("User-Agent"), r.hasher.Hash(req))
 	if resp != nil {
 		resp.Header.Set(CacheModeHeader, string(cacheMode))
+		if cacheMode == ModeRevalidated && resp.Header.Get(cacheEntryCreationDateHeader) != "" {
+			intVal, err := strconv.Atoi(resp.Header.Get(cacheEntryCreationDateHeader))
+			if err != nil {
+				logrus.WithError(err).WithField("header-value", resp.Header.Get(cacheEntryCreationDateHeader)).Warn("Failed to convert cacheEntryCreationDateHeader value to int")
+			} else {
+				ghmetrics.CollectCacheEntryAgeMetrics(float64(time.Now().Unix()-int64(intVal)), req.URL.Path, req.Header.Get("User-Agent"), r.hasher.Hash(req))
```

Comment:
Why not `time.Unix(intVal,0)` and  `time.Sub()` to get the differences? It's important not to have inaccurate metrics on leap second days :trollface: